orga/app/styles/globals/buttons.scss

```
@@ -1,4 +1,6 @@
 .button {
+  box-sizing: border-box;
+  font-family: "Roboto";
```

Il vaut mieux utiliser la variable sass (car elle a les polices "fallback" au cas où Roboto n'est pas dispo)
```suggestion
  font-family: $roboto;
```